src/content/docs/style-guide/writing-strategies/Untitled-1

```diff
@@ -0,0 +1,29 @@
+---
+title: OpenTelemetry examples
+metaDescription: "Our stance on OpenTelemetry examples moving forward."
+---
+
+This page outlines the docs team's approach to documenting OpenTelemetry examples.
+
+## Background [#background]
+
+In New Relic documentation for OpenTelemetry (OTEL), we provide examples aimed to demonstrate the most common configurations we expect users to use. Historically, the docs site contained these examples on the docs site. However, in May 2024, a joint effort with Relics working on OTEL, we decided to migrate these examples from the docs site to the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main) repo. For example, here’s a PR where we moved all the infrastructure examples: [PR #17742](https://github.com/newrelic/docs-website/pull/17742).
```

Comment:
Even though the style guide is quasi-internal, I think it's best to not use more internal terms like "Relics". How about "a joint effort between the docs and OTel team..."

Also, is OTEL always all caps like that?